tests/testutil/service.go

```diff
@@ -114,6 +130,27 @@ func (ts *TestService) Weight(percentage int) {
 	}
 }
 
+// RolloutWeight calls "rio weight --rollout=true --rollout-increment={increment} --rollout-interval={interval} ns/service:version={percentage}" on this service
+func (ts *TestService) RolloutWeight(increment int, interval int, percentage int) {
+	// Set increment and interval to their default values if a negative number is passed in
+	if increment < 0 {
+		increment = 5
+	}
+	if interval < 0 {
+		interval = 5
+	}
```

Comment:
I don't see the point of this logic, if we want to normalize the values that should be done in the actual code. Also the tests are only ones passing this info in, no user input here.